core/src/main/scala/kanaloa/reactive/dispatcher/queue/Worker.scala

```diff
@@ -195,18 +187,22 @@ trait Worker extends Actor with ActorLogging with MessageScheduler {
     startAt:       LocalDateTime = LocalDateTime.now
   ) {
     def success(result: Any): Unit = {
```

Comment:
to me inner class is like nested function, it's natural to me that they access parent state. This is old code though, so how about we address this issue in another PR?